internal/unionstore/memdb.go

```diff
@@ -155,6 +155,20 @@ func (db *MemDB) Cleanup(h int) {
 	db.stages = db.stages[:h-1]
 }
 
+type MemCheckpoint = memdbCheckpoint
```

Comment:
How about changing `memdbCheckpoint` to public?

Reply:
ok, done.